app/helpers/products_helper.rb

```diff
@@ -106,4 +106,12 @@ def with_c3_task(task)
     end
     [task]
   end
+
+  def provider_npi(provider)
```

Comment:
so if the provider model is in health-data-standards, should we just extend the provider model in cypress?